infra/terraform/platform/main.tf

```diff
@@ -205,3 +205,27 @@ module "buckets_archiver" {
     "component", "buckets-archiver",
   ), var.tags)}"
 }
+
+module "kubernetes_master_monitoring" {
+  source = "../modules/elb_cloudwatch_alerts"
+
+  name          = "${terraform.workspace}-kubernetes-master-alerts"
+  elb_name      = "${var.kubernetes_master_elb_name}"
```

Comment:
I see in the variables that this value is `api-dev`/`api-alpha`, potentially you could get rid of the `kubernetes_master_elb_name ` variable in the tfvar file and use `elb_name      = "api-${terraform.workspace}"` here.

Similar below you could get rid of the `bastion_elb_name` variable and replace with `elb_name      = "bastion-${terraform.workspace}"`.

If you decide to not do this is also fine, I would personally slightly prefer to have 2 variables less in this case but it's not the end of the world anyway ☺️ .

Reply:
Yes good idea. I'll change them to use that